kubeshop/testkube-dashboard
# Delete unused code

## Changes

- Delete unused code
   - I left a few unused parts that may be helpful later - some types, API calls, short form in `displayTimeBetweenDates`
   - I avoided modifying the code, only deleting
- Besides `package-lock` changes, there are **17 additions, 912 deletions**

## Fixes

- https://github.com/kubeshop/testkube/issues/3628

## How to test it

- Build application (to check types) and go through it (to see if nothing really breaks)

## screenshots

-

## Checklist
